downloads/swan-lake-release-notes/swan-lake-beta4/RELEASE_NOTE.md

```diff
@@ -789,16 +789,16 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Beta4](https://github
 - Added `error:StackFrame[]` as a key-value pair type
     
 #### `http` Package
-- Marked the HTTP service type as distinct
 - Relaxed the data-binding restriction for status codes without content
 - Changed the `Listener.getConfig()` API to return an `InferredListenerConfiguration`
 
 #### `websub` Package
 - Updated to not change the generated unique-service-path after compilation
-- Marked the WebSub service type as distinct
 
-#### `websubhub` Package
-- Marked the WebSubHub service type as distinct
+### Changes
```

Comment:
```suggestion
### Changes
```
IMHO, just saying `Changes` is not meaningful enough. Can't we add this under the 3 topics in the current structure, preserving consistency without introducing a new level 3 topic? Also, if this is related to all the modules, shall we add a topic such as `All Packages` under `Improvements`?